src/web/app/components/question-types/question-edit-details-form/rubric-question-edit-details-form.component.html

```diff
@@ -8,69 +8,73 @@
     </div>
   </div>
 </div>
-<table id="rubric-table" class="table table-bordered">
-  <tbody>
-    <tr class="fw-bold item-center">
-      <td>
-        <div>Choices <i class="fas fa-arrow-right"></i></div>
-      </td>
-      <td *ngFor="let rubricChoice of model.rubricChoices; let j = index; trackBy: trackByIndex" [ngClass]="{'highlight-deletion': j === columnToHighlight}">
-        <input type="text" class="form-control fw-bold" [disabled]="!isEditable"
-               [ngModel]="rubricChoice" (ngModelChange)="triggerRubricChoiceChange($event, j)">
-      </td>
-    </tr>
-    <tr *ngFor="let rubricDescriptionRow of model.rubricDescriptions; let i = index; trackBy: trackByIndex" [ngClass]="{'highlight-deletion': i === rowToHighlight}">
-      <td>
-        <div class="input-group">
-          <div class="btn-group-vertical">
-            <button type="button" class="btn btn-light" [disabled]="i === 0 || !isEditable"
-                    ngbTooltip="Move row up" (click)="moveRow(i, i - 1)">
-              <i class="fas fa-arrow-up"></i>
+<div class="table-responsive">
+  <table id="rubric-table" class="table table-bordered">
+    <thead>
+      <tr class="fw-bold item-center">
+        <td>
+          <div>Choices <i class="fas fa-arrow-right"></i></div>
+        </td>
+        <th *ngFor="let rubricChoice of model.rubricChoices; let j = index; trackBy: trackByIndex" [ngClass]="{'highlight-deletion': j === columnToHighlight}" scope="col">
+          <input type="text" class="form-control fw-bold" [disabled]="!isEditable"
+                 [ngModel]="rubricChoice" (ngModelChange)="triggerRubricChoiceChange($event, j)">
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let rubricDescriptionRow of model.rubricDescriptions; let i = index; trackBy: trackByIndex" [ngClass]="{'highlight-deletion': i === rowToHighlight}">
+        <th scope="row" [attr.aria-label]="model.rubricSubQuestions[i]" class="row-header">
+          <div class="input-group">
+            <div class="btn-group-vertical">
+              <button type="button" class="btn btn-light" [disabled]="i === 0 || !isEditable"
+                      ngbTooltip="Move row up" (click)="moveRow(i, i - 1)">
+                <i class="fas fa-arrow-up"></i>
+              </button>
+              <button id="btn-delete-subquestion" type="button" class="btn btn-light" (click)="deleteSubQuestion(i)"
+                      (mouseover)="rowToHighlight = i" (mouseleave)="rowToHighlight = -1"
+                      [disabled]="model.rubricSubQuestions.length === 1 || !isEditable">
+                <i class="fas fa-times"></i>
+              </button>
+              <button type="button" class="btn btn-light" [disabled]="i === this.model.rubricSubQuestions.length - 1 || !isEditable"
+                      ngbTooltip="Move row down" (click)="moveRow(i, i + 1)">
+                <i class="fas fa-arrow-down"></i>
+              </button>
+            </div>
```

Comment:
Think can add `aria-label`s for these buttons? Else the screen reader just reads it as "button". If I'm not wrong, shouldn't be too complicated, just use the same text as in the `ngbTooltip` / "Delete row".